src/redisai.h

```diff
@@ -54,16 +60,18 @@ void MODULE_API_FUNC(RedisAI_ScriptRunCtxFree)(RAI_ScriptRunCtx* mctx);
 int MODULE_API_FUNC(RedisAI_ScriptRun)(RAI_ScriptRunCtx* mctx);
 RAI_Script* MODULE_API_FUNC(RedisAI_ScriptGetShallowCopy)(RAI_Script* model);
 
+int MODULE_API_FUNC(RedisAI_GetAPIVersion)();
+
 
 #define REDISAI_MODULE_INIT_FUNCTION(name) \
   if (RedisModule_GetApi("RedisAI_" #name, ((void **)&RedisAI_ ## name))) { \
     printf("could not initialize RedisAI_" #name "\r\n");\
-    return false; \
+    return REDISMODULE_ERR; \
   }
 
-static bool RediDL_Initialize(){
+static int RediAI_Initialize(){
```

Comment:
This should be `RedisAI_Initialize` (missing `s`)

Reply:
fixed, thanks.